apps/dotcom/client/src/tla/app/TldrawApp.ts

```diff
@@ -95,7 +96,7 @@ export class TldrawApp {
 				})
 				const token = await getToken()
 				params.set('accessToken', token || 'no-token-found')
-				return `${process.env.MULTIPLAYER_SERVER}/api/app/${userId}/connect?${params}`
+				return `${MULTIPLAYER_SERVER}/api/app/${userId}/connect?${params}`
```

Comment:
```suggestion
				return `${MULTIPLAYER_SERVER}/app/${userId}/connect?${params}`
```

Looks like `MULTIPLAYER_SERVER` already adds the `/api`